mkdocs_static_i18n/plugin.py

```diff
@@ -422,6 +423,9 @@ def on_page_context(self, context, page, config, nav):
                 alternate["link"] += f"{separator}{page_url}"
             config["extra"]["alternate"] = alternates
 
+        if self.config["site_name_translations"].get(page.file.dest_language, {}):
```

Comment:
```suggestion
        if self.config["site_name_translations"].get(page.file.dest_language):
```

I'm not sure adding a default of `{}` is useful on a if block like this one